PatchLoader/PatchScanner.cs

```diff
@@ -17,22 +17,76 @@ public PatchScanner(Logger logger)
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
         }
 
-        public List<KeyValuePair<string, IPatch>> Scan(string path) {
+        public List<KeyValuePair<string, IPatch>> Scan(IEnumerable<string> paths) {
+            bool ignoreExcluded = false;
+            List<KeyValuePair<string, List<KeyValuePair<ModDirStatus, string>>>> directoriesPerPath = new List<KeyValuePair<string, List<KeyValuePair<ModDirStatus, string>>>>();  
```

Comment:
```List<KeyValuePair<string, List<KeyValuePair<ModDirStatus, string>>>>```
🤣

Reply:
making an exception for `LoadOrder.ipatch` results in interesting code.

Reply:
you know, I was a bit lazy and I didn't see enough benefit for introducing new type/struct in a method that is just a wrapper so... it is what it is. Works 🙃